lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/persistence/SQLiteLoader.java

```diff
@@ -244,19 +251,30 @@ private void initTables() throws SQLException {
         statement.executeUpdate("CREATE TABLE TrafficSignals (id STRING, ref_node_id STRING, phases STRING, timing STRING, from_way_id STRING, via0_way_id STRING, via1_way_id STRING, to_way_id STRING, lanes_from STRING, lanes_via0 STRING, lanes_via1 STRING, lanes_to STRING)");
         // vehicle data
         statement.executeUpdate("CREATE TABLE Route (id STRING, sequence_number INTEGER, connection_id STRING)");
-        // buildings
-        statement.executeUpdate("CREATE TABLE Corner (id STRING, lat DOUBLE, lon DOUBLE, x DOUBLE, y DOUBLE)");
-        statement.executeUpdate("CREATE TABLE Wall (id STRING, building_id STRING, from_corner_id STRING, to_corner_id STRING, length DOUBLE, sequence_number INTEGER)");
-        statement.executeUpdate("CREATE TABLE Building (id STRING, name TEXT, height DOUBLE, min_x DOUBLE, max_x DOUBLE, min_y DOUBLE, max_y DOUBLE)");
         // roundabouts
         statement.executeUpdate("CREATE TABLE Roundabout (id STRING)");
         statement.executeUpdate("CREATE TABLE RoundaboutConsistsOf (roundabout_id STRING, node_id STRING, sequence_number INTEGER)");
+        // buildings
+        statement.executeUpdate("CREATE TABLE Building (id STRING, name TEXT, height DOUBLE)");
+        statement.executeUpdate("CREATE TABLE BuildingCorner (building_id STRING, lat DOUBLE, lon DOUBLE, sequence_number INTEGER)");
```

Comment:
I understand the new db / table design. Yet, this needs to be documented.

`BuildingCorner` is ok for me, we might consider `BuildingConsistsOf` similar to roundabouts

Reply:
`BuildingConsistsOf` fits better, thanks